scripts/modelbench/bisect_run_test.sh

```diff
@@ -43,6 +43,21 @@ run_perf_drop_test() {
     fi
 }
 
+run_perf_improve_test() {
+    detected_value=$(bash ./inductor_single_run.sh $THREADS $MODE $SCENARIO $SUITE $MODEL $DT $CHANNELS $SHAPE $WRAPPER $BS $FREEZE | tail -n 1 | awk -F, '{print $5}')
+    result=$(echo $detected_value | awk '{ printf "%.5f", $1/1000 }')
+    echo "=====result: $result======="
+    ratio=$(echo "$EXP_PERF $result" | awk '{ printf "%.2f\n", $1/$2 }')
+    echo "=====ratio: $ratio======="
+    if (( $(echo "$ratio > $PERF_RATIO" | bc -l) )); then
+	    echo "`git rev-parse HEAD` is a BAD COMMIT!"
```

Comment:
We can name it `GOOD` commit, else `BAD` commit to avoid confuse.
Also change the log in https://github.com/chuanqi129/inductor-tools/pull/46/files#diff-d6e0dc8e86226f4799b14e2b4884673e94ac5aeb2fc5b46f3538d422dd7ea944R68 to `first bad/good commit`